src/pasteInlineForm.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react'
+import {
+  Form,
+  FormGroup,
+  FormControl,
+  Button
+} from 'react-bootstrap'
+
+class PasteInlineForm extends Component {
+  render () {
+    console.log(this.props)
```

Comment:
Should not have any `console.log()` entries in your code. Is this passing lint? Your build should not be passing if linting is failing.